src/components/YourComponent.js

```diff
@@ -0,0 +1,35 @@
+// src/components/YourComponent.js
+
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook, removeBook } from '../redux/books/booksSlice';
+
+const YourComponent = () => {
+  const dispatch = useDispatch();
+
+  const handleAddBook = () => {
+    const newBook = {
+      item_id: 'item4', // Replace with a unique ID generation logic
+      title: 'New Book',
+      author: 'Author Name',
+      category: 'Fiction', // Replace with the actual category
+    };
+
+    dispatch(addBook(newBook));
+  };
+
+  const handleRemoveBook = (bookId) => {
+    dispatch(removeBook(bookId));
+  };
+
+  return (
+    <div>
+      {/* Your component content */}
+      <button type="button" onClick={handleAddBook}>Add Book</button>
+      <button type="button" onClick={() => handleRemoveBook('item1')}>Remove Book</button>
+      {/* Replace 'item1' with the actual ID of the book you want to remove */}
+    </div>
+  );
```

Comment:
Great job @Yonasle 🤝

- [ ] Good job implementing the add and delete buttons logic. However, these buttons are not being rendered. Could you make sure these buttons are rendered on the page? 
![buttons](https://github.com/Yonasle/bookstore_react/assets/63917725/1a00e33c-0724-43fa-b850-ac7efa1e4d18)